docs/topics/async.txt

```diff
@@ -63,81 +63,88 @@ deploy Django using :doc:`ASGI </howto/deployment/asgi/index>` instead.
     on debug logging for the ``django.request`` logger and look for log
     messages about *`"Synchronous middleware ... adapted"*.
 
-In either ASGI or WSGI mode, though, you can safely use asynchronous support to
-run code in parallel rather than serially, which is especially handy when
-dealing with external APIs or datastores.
+In both ASGI or WSGI mode, you can still safely use asynchronous support to
+run code in parallel rather than serially. This is especially handy when
+dealing with external APIs or data stores.
 
-If you want to call a part of Django that is still synchronous (like the ORM)
-you will need to wrap it in a :func:`sync_to_async` call, like this::
+If you want to call a part of Django that is still synchronous, like the ORM,
+you will need to wrap it in a :func:`sync_to_async` call. For example::
 
     from asgiref.sync import sync_to_async
 
-    results = sync_to_async(MyModel.objects.get)(pk=123)
+    results = sync_to_async(Blog.objects.get)(pk=123)
 
 You may find it easier to move any ORM code into its own function and call that
-entire function using :func:`sync_to_async`. If you accidentally try to call
-part of Django that is still synchronous-only from an async view, you will
-trigger Django's :ref:`asynchronous safety protection <async-safety>` to
-protect your data from corruption.
+entire function using :func:`sync_to_async`. For example::
+
+    from asgiref.sync import sync_to_async
+
+    @sync_to_async
+    def get_blog(pk):
+        return Blog.objects.select_related('author').get(pk=pk)
+
+If you accidentally try to call part of Django that is still synchronous-only
+from an async view, you will trigger Django's
+:ref:`asynchronous safety protection <async-safety>` to protect your data from
+corruption.
 
 Performance
 -----------
 
-When running in a mode that does not match the view (e.g. an async view under
+When running in a mode that does not match the view (i.e. an async view under
```

Comment:
Is this the correct use of i.e.? I would have thought this remained an e.g.

Reply:
"e.g." is for an example out of a set, but the list here is exhaustive (right?). Therefore it's an "i.e.", no?

Reply:
Hard to say really - you could also include things like "an async view under the sync test client" or "sync middleware under ASGI", but those maybe aren't the same list?

Reply:
Ok reverted.